src/main/java/core/basesyntax/service/ConvertorService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ConvertorService {

```

Comment:
just better name suggestion
```suggestion
public interface ParserService {

```